Makefile

```
@@ -62,7 +62,7 @@ build: ui build-all-go ## Build all components
 build-all-go: bin/vmclarity-apiserver bin/vmclarity-cli bin/vmclarity-orchestrator bin/vmclarity-ui-backend bin/vmclarity-cr-discovery-server ## Build all go components
 
 BUILD_OPTS = -race
-BUILD_OPTS += -tags="$(GO_BUILD_TAGS)"
+BUILD_OPTS += -tags=$(call subst-space-with-comma,$(GO_BUILD_TAGS))
```

```suggestion
ifneq ($(strip $(GO_BUILD_TAGS)),)
	BUILD_OPTS += -tags=$(call subst-space-with-comma,$(GO_BUILD_TAGS))
endif
```

To make sure we only append go build tags if they are set.

---

Right now we always have two, but I'll add it for extra safety in case those will be removed in the future